hack/update-deps.sh

```diff
@@ -13,11 +13,11 @@ set -o pipefail
 cd "${ROOT_DIR}"
 
 # This controls the knative release version we track.
-KN_VERSION="release-1.2"
+KN_VERSION="release-1.3"
 EVENTING_VERSION="release-v1.2"
 EVENTING_KAFKA_VERSION="release-v1.2"
 EVENTING_KAFKA_BROKER_VERSION="release-v1.2"
-SERVING_VERSION="release-v1.2.0"
+SERVING_VERSION="release-v1.3"
```

Comment:
IMHO, dependencies should be bumped for every component together and we can remove replace directives.

Reply:
I am in favor of this but I think the last time we didnt want to do a big PR for all deps.

Reply:
Yes as I mentioned I prefer that, I will bump Eventing too. I added the replacements so Eventing can do the rest of the bumping anyway.